main.py

```diff
@@ -381,6 +401,211 @@ def job_posting(username):
+    print("\nThere are no jobs you've applied for.")
+    choose_features(username)
+
+
+def list_unapplied_jobs(username):
+  """job selection page"""
+  draw_line(message="LIST_UNAPPLIED_JOBS") 
+  #Get all jobs from database
+  applied_jobs = applied_jobs_list(username)
+
+  if applied_jobs != False:
+    s = set(applied_jobs)
+    unapplied_jobs = [job for job in all_jobs_list(username) if job not in s]
+
+    #Print all jobs and prompt user if they wish to apply to any of the jobs listed
+    if unapplied_jobs:
+      print("\nListing all jobs you have NOT applied for:")
+      print(f"\n{', '.join(unapplied_jobs)}")
+      print("\n")
+      choose_features(username)
+
+    else:
+      print("\nThere are no jobs you have NOT applied for.")
+      choose_features(username)
+  else:
+    all_jobs = all_jobs_list(username)
+    print("\nListing all jobs you have NOT applied for:")
+    print(f"\n{', '.join(all_jobs)}")
+    print("\n")
+    choose_features(username)
+
+  
+
+#Function designed to select a job to apply (list all available jobs)
+def job_select(username):
+  """job selection page"""
+  draw_line(message="JOB_SELECT") 
+  #Get all jobs from database
+  all_jobs = all_jobs_list(username)
+  
+  #Print all jobs and prompt user if they wish to apply to any of the jobs listed
+  if all_jobs != False:
+    print("\nHere are all jobs available:")
+    print(f"\n{', '.join(all_jobs)}")
+    print("\n")
+    view_info = input(
+        "Do you want to apply to any of the jobs on this list? y/n?: "
+    ).lower()
+
+    #If user selects yes, have them search for the job
+    if view_info == "y" or view_info == "Y":
+        apply_for_job(username)
+      
+    
+    #If you select no or other options, then prompt user to go back to feature select
+    else:
+        if go_back():
+            choose_features(username)
+  #Else, upon a failed search, inform the user that no account has that last name
+  else:
+    print(
+        "\nThere are no jobs opening on inCollege."
+    )
+    
+
+#Function sends friend request out to a specific user
+def apply_for_job(username):
+
+    draw_line(message="JOB_CONFIRM") 
+    """Promopt the user to search by job title , inform them, and ask for confirmation"""
+    job_title = input("\nEnter the title of the job you want to apply for: ")
+
+    #Check if the user exists
+    job_info = get_job(job_title)
+
+    #If the user exists, inform the user that the request has been sent
+    if job_info:
+        print("\nThis is the current job information for this title:")
+        print(f"\nTitle: {job_info[0]}")
+        print(f"Description: {job_info[1]}")
+        print(f"Employer: {job_info[2]}")
+        print(f"Location: {job_info[3]}")
+        print(f"Salary: {job_info[4]}")
+        print("\n")
+
+        
+
+        confirm_apply = input(
+          "Confirm this job and send the application? y/n?: "
+        ).lower()
+
+        #If user selects yes, peform the send application function

```

Comment:
here, you are making confirm_apply to be always in lowercase. so there is no need to check if it uppercase